pkg/scheduler/framework/plugins/interpodaffinity/interpod_affinity.go

```diff
@@ -20,47 +20,133 @@ import (
 	"context"
 	"fmt"
 
-	v1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+
```

Comment:
they should be in the same block. Although I think we are not being consistent throughout the code yet.